airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py

```diff
@@ -159,26 +152,25 @@ def test_default_start_date():
     [
         (TEST_CONFIG, "aW50ZWdyYXRpb24tdGVzdEBhaXJieXRlLmlvL3Rva2VuOmFwaV90b2tlbg=="),
         (TEST_CONFIG_OAUTH, "test_access_token"),
-        (TEST_OLD_CONFIG, "aW50ZWdyYXRpb24tdGVzdEBhaXJieXRlLmlvL3Rva2VuOmFwaV90b2tlbg=="),
     ],
-    ids=["api_token", "oauth", "old_config"],
+    ids=["api_token", "oauth"],
 )
 def test_get_authenticator(config, expected):
     # we expect base64 from creds input
-    result = SourceZendeskSupport().get_authenticator(config=config)
+    result = SourceZendeskSupport(config=config, catalog=None, state=None).get_authenticator(config=config)
     assert result._token == expected
 
 
 @pytest.mark.parametrize(
     "response, start_date, check_passed",
-    [({"active_features": {"organization_access_enabled": True}}, "2020-01-01T00:00:00Z", True), ({}, "2020-01-00T00:00:00Z", False)],
+    [([{"active_features": {"organization_access_enabled": True}}], "2020-01-01T00:00:00Z", True), ([], "2020-01-01T00:00:00Z", False)],
```

Comment:
Changed two things:
* Mocked `UserSettingsStream.read_records` instaed of `UserSettingsStream.get_settings` to have better test coverage
* Fixed the datetime `2020-01-00T00:00:00Z` that is not a valid date

Reply:
`2020-01-00T00:00:00Z` == `2019-12-31T00:00:00Z`. everybody knows this